tests/base.py

```diff
@@ -300,13 +301,13 @@ def __init__(self):
         self.shot_code      = None
         self.task_content   = None
 
-
     def read_config(self, config_path):
         config_parser = ConfigParser()
         config_parser.read(config_path)
         for section in config_parser.sections():
             for option in config_parser.options(section):
-                value = config_parser.get(section, option)
+                value = (os.environ.get('SG_{0}'.format(str(option).upper())) or
```

Comment:
I'm breaking this up a bit more in the next commit, since getting the environment variables here is dependent on whether someone has setup a config file properly (or has one at all).